.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -45,14 +45,26 @@ body:
     attributes:
       label: Screenshots
       description: If applicable, add screenshots to help explain your problem.
+  - type: markdown
+    attributes:
+      value: |-
+        ## Latest stable version of Heroic is 2.5.2. Please, update Heroic before reporting a bug if you have an older version.
```

Comment:
One will need to update this line every time a new stable and beta version gets released.

Reply:
That's a good point. Although not a blocker, just one more thing to prepare when doing a new release, ahahaha